Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Author Cameron-git
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive 
+
+WORKDIR /app
+
+RUN apt-get update
+RUN apt-get upgrade -y
```

Comment:
Apparently this isn't best practice since if you build the image again later it might be different. I'm not sure this is a massive problem in our case though.

Reply:
Good point, but Ubuntu should be pretty stable, especially LTS versions (20.04 is LTS)